Scoop/NetworkManager.swift

```diff
@@ -10,15 +10,120 @@ import Alamofire
 
 class NetworkManager {
     
+    var currentUser = User() //Swap with BaseUser once backend finishes profile routes
+    
+    static var userToken = Constants.UserDefaults.accessToken
+    
     static let shared: NetworkManager = NetworkManager()
     
-    private static let hostEndpoint = "https//:\(Keys.scoopedServer)"
+    private static let hostEndpoint = "https://\(Keys.scoopedServer)"
     
     static var headers: HTTPHeaders {
-            let accessToken = UserDefaults.standard.string(forKey: Constants.UserDefaults.accessToken) ?? ""
+            let accessToken = userToken
             let headers: HTTPHeaders = [
                 "Authorization": "Token \(accessToken)"
             ]
             return headers
         }
+    
+    static func authenticateUser(googleID: String, email: String, firstName: String, lastName: String, id_token: String, completion: @escaping (Result<UserSession, Error>) -> Void) {
+        let parameters: [String : String] = [
+            "sub": googleID,
+            "email": email,
+            "given_name": firstName,
+            "family_name": lastName,
+            "id_token": id_token
+        ]
+        AF.request("\(hostEndpoint)/api/authenticate/", method: .post, parameters: parameters, encoding: JSONEncoding.default).validate().responseData { response in
+            switch response.result {
+            case .success(let data):
+                let jsonDecoder = JSONDecoder()
+                jsonDecoder.keyDecodingStrategy = .convertFromSnakeCase
+                
+                do{
```

Comment:
`do {`

Reply:
Whoops, didn't see this one, fixed on next PR

Reply:
Can you address this now?